users.json

```diff
@@ -0,0 +1,2 @@
+{"_id":{"$oid":"5de6642a30601d3f60ca27fb"},"username":"abc","password":"123"}
```

Comment:
don't store passwords in the DB
https://stackoverflow.com/questions/1054022/best-way-to-store-password-in-database